test/parallel/test-buffer-arraybuffer.js

```diff
@@ -43,12 +43,16 @@ assert.throws(function() {
   Buffer.from(new AB());
 }, TypeError);
 
-// write{Double,Float}{LE,BE} with noAssert should not crash, cf. #3766
+// https://github.com/nodejs/node/issues/3766
 const b = Buffer.allocUnsafe(1);
-b.writeFloatLE(11.11, 0, true);
-b.writeFloatBE(11.11, 0, true);
-b.writeDoubleLE(11.11, 0, true);
-b.writeDoubleBE(11.11, 0, true);
+assert.throws(() => b.writeFloatLE(11.11, 0),
+              /RangeError: out of range index/);
```

Comment:
I think these four tests can be removed if you want, as they were a regression test for a crash when `noAssert` was `true`.  No `noAssert` means no regression possible. :-D